internal/authorizedapp/database/authorized_app.go

```diff
@@ -37,6 +37,43 @@ func NewAuthorizedAppDB(db *database.DB) *AuthorizedAppDB {
 	}
 }
 
+func (aa *AuthorizedAppDB) GetAllAuthorizedApps(ctx context.Context, sm secrets.SecretManager) ([]*model.AuthorizedApp, error) {
+	conn, err := aa.db.Pool.Acquire(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("acquiring connection: %v", err)
+	}
+	defer conn.Release()
+
+	query := `
+    SELECT
+      app_package_name, platform, allowed_regions,
+      safetynet_apk_digest, safetynet_cts_profile_match, safetynet_basic_integrity, safetynet_past_seconds, safetynet_future_seconds,
```

Comment:
I think we want the new fields for devicecheck_disabled and safetnet_disabled in here

Reply:
got that in a local rebase.

Reply:
rebased after your change went in - have local.